packages/components/src/List/Toolbar/DisplayModeToggle/DisplayModeToggle.component.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { ActionIconToggle } from '../../../Actions';
+import getDefaultT from '../../../translate';
+
+import theme from './DisplayModeToggle.scss';
+
+const options = ['table', 'large'];
+function getLabel(selected, t) {
+	switch (selected) {
+		case 'table':
+			return t('LIST_SELECT_DISPLAY_MODE_TABLE', { defaultValue: 'Table' });
+		case 'large':
+			return t('LIST_SELECT_DISPLAY_MODE_LARGE', { defaultValue: 'Expanded' });
+		default:
+			return t('LIST_SELECT_DISPLAY_MODE_TABLE', { defaultValue: 'Table' });
+	}
+}
+
+function DisplayModeToggle({ id, displayModes, onChange, mode, t }) {
+	const modes = displayModes || options;
+
+	function getActionIcon(option) {
+		return (
+			<ActionIconToggle
+				key={option}
+				id={`${id}-${option}`}
+				icon={option === 'table' ? 'talend-table' : 'talend-expanded'}
+				label={getLabel(option, t)}
+				aria-label={t('LIST_SELECT_DISPLAY_MODE', {
```

Comment:
This doesn't seem to be passed to the button. The label is taken into account as tooltip and read by SR.
Can you remove that and set the message as label prop instead.

```
const label = getLabel(option, t); (BTW this should return option as default, not Table)
<ActionIconToggle
	label={t('LIST_SELECT_DISPLAY_MODE', {
		defaultValue: 'Set {{displayMode}} as current display mode.',
		displayMode: label,
	})}
```

Reply:
Still KO